.github/workflows/ci_tests.yaml

```diff
@@ -91,22 +91,42 @@ jobs:
           path: |
             ~/.gmt/cache
             ~/.gmt/server
-          key: cache-gmt-${{ runner.os }}-${{ github.ref }}-20200711
+          key: cache-gmt-${{ runner.os }}-${{ github.ref }}-20200715
           restore-keys: cache-gmt-${{ runner.os }}-refs/heads/master-
 
       # Workaround for the timeouts of 'gmt which' on Linux and Windows
       - name: Download remote data using wget (Linux & Windows)
         shell: bash -l {0}
         run: |
           if [ "$RUNNER_OS" == "Windows" ]; then choco install wget; fi  # install wget on Windows
-          mkdir -p ~/.gmt ~/.gmt/cache ~/.gmt/server/earth/earth_relief/
-          wget --no-check-certificate https://oceania.generic-mapping-tools.org/gmt_hash_server.txt -P ~/.gmt/server/
-          for data in earth_relief_01d_p.grd earth_relief_01d_g.grd earth_relief_30m_p.grd earth_relief_30m_g.grd earth_relief_10m_p.grd earth_relief_10m_g.grd; do
-            wget --no-check-certificate https://oceania.generic-mapping-tools.org/server/earth/earth_relief/${data} -P ~/.gmt/server/earth/earth_relief/
+          GMT_DATA_SERVER=https://oceania.generic-mapping-tools.org
+          wget_input="wget_input.txt"
+          data="earth_relief_01d_p.grd earth_relief_01d_g.grd earth_relief_30m_p.grd earth_relief_30m_g.grd earth_relief_10m_p.grd earth_relief_10m_g.grd"
+          cache="ridge.txt Table_5_11.txt test.dat.nc tut_bathy.nc tut_quakes.ngdc tut_ship.xyz usgs_quakes_22.txt"
+
+          echo "${GMT_DATA_SERVER}/gmt_hash_server.txt" > $wget_input
+          echo "${GMT_DATA_SERVER}/gmt_data_server.txt" >> $wget_input
+          for file in $data; do
+            echo "${GMT_DATA_SERVER}/server/earth/earth_relief/$file" >> $wget_input
           done
-          for data in ridge.txt Table_5_11.txt test.dat.nc tut_bathy.nc tut_quakes.ngdc tut_ship.xyz usgs_quakes_22.txt; do
-            wget --no-check-certificate https://oceania.generic-mapping-tools.org/cache/${data} -P ~/.gmt/cache/
+          for file in $cache; do
+            echo "${GMT_DATA_SERVER}/cache/$file" >> $wget_input
           done
+
+          mkdir cachedir
+          wget --no-check-certificate -i $wget_input -P cachedir
```

Comment:
It would be nice to just put the `wget_input.txt` file somewhere (e.g. in `.github/workflows`) instead of making one from scratch. But I see that we need the `$data` and `$cache` variables.